Lost Together/Assets/Scripts/EnemyPatrol.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyPatrol : MonoBehaviour
+{
+    public Transform pointA;
+    public Transform pointB;
+    public float speed = 2f;
+    public float chaseRange = 5f;
+    private Rigidbody2D rb;
+    private Animator anim;
+    private Transform currentPoint;
+    private GameObject[] players;
+
+    private void Start()
+    {
+        rb = GetComponent<Rigidbody2D>();
+        anim = GetComponent<Animator>();
+        currentPoint = pointB; 
+        anim.SetBool("isRunning", true);
+    }
+
+   private void Update()
+{
+    GameObject[] players = GameObject.FindGameObjectsWithTag("Player1");
```

Comment:
Line 26 and 27, why do you assign those as GameObject arrays? We know we always have only one Player1 and Player2 object in the scene so I think you can refactor this.